kolibri/core/auth/utils.py

```diff
@@ -10,3 +13,41 @@ def confirm_or_exit(message):
     if answer != "yes":
         print("Canceled! Exiting without touching the database.")
         sys.exit(1)
+
+
+def get_facility(facility_id=None, noninteractive=False):
+    # try to get a valid facility from id
+    if facility_id:
+        try:
+            facility = Facility.objects.get(id=facility_id)
+        except Facility.DoesNotExist:
+            raise CommandError("Facility with ID {} does not exist".format(facility_id))
+    # if no id passed in, assume only one facility on device
+    else:
+        try:
+            facility = Facility.objects.get()
+        except Facility.DoesNotExist:
+            raise CommandError(
+                (
+                    "There are no facilities on this device. "
+                    "Please setup your kolibri instance."
```

Comment:
Perhaps being clearer in how to do so -- "Please initialize your Kolibri installation by starting the server, loading it in the browser, and following the instructions" or something.